packages/core/src/cloud-wallet.ts

```diff
@@ -234,6 +234,7 @@ export async function initializeCloudWallet({
   async function clearEdvDocuments() {
     const allDocs = await storageInterface.find({});
 
+    debugger;
```

Comment:
can be removed?

Reply:
it was removed already, should be fine now